mono/dis/get.c

```diff
@@ -1864,6 +1866,7 @@ get_memberref_container (MonoImage *m, guint32 mrp_token, MonoGenericContainer *
 	guint32 table = mrp_token & 7;
 	guint32 idx = mrp_token >> 3;
 
+	error_init (error);
```

Comment:
We should initialize where we create it (even without `ERROR_DECL`) since it's the way we are going towards.

Reply:
Good point.  I'll update to the same semantics as #6406 and wait for it to merge.